src/progressbar/progressbar.spec.ts

```diff
@@ -0,0 +1,116 @@
+import {iit, it, ddescribe, describe, expect, injectAsync, TestComponentBuilder} from 'angular2/testing';
+
+import {Component} from 'angular2/angular2';
+
+import {NgbProgressbar} from "./progressbar";
+
+function getBarWidth(nativeEl): number {
+  return nativeEl.querySelector('.progress-bar').style.width;
+}
+
+function getProgressbar(nativeEl: Element): Element {
+  return nativeEl.querySelector('progress');
+}
+
+describe('ng-progressbar', () => {
+
+  describe('business logic', () => {
```

Comment:
There, but it needs extra logic to be done and better if we start with a smaller PR with an initial implementation and then we move to add more features. That is my next one.